startup.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+
+if [[ -z "${CLOUDSQL_PROXY_SA}" ]]; then
+    echo "'CLOUDSQL_PROXY_SA' environment variable not defined. Place the contents of the service account json in this environment variable" 1>&2
+    exit 1
+else
+    # Replace \\n with an actual newline, for .env file compatibility
+    #echo "${CLOUDSQL_PROXY_SA//\\n/$'\n'}" > /sa.json
+    echo "${CLOUDSQL_PROXY_SA}" > /sa.json
+fi
+
+if [[ -z "${GCLOUD_SQL_INSTANCE}" ]]; then
+    echo "'GCLOUD_SQL_INSTANCE' environment variable not defined. Specify the cloudsql instance in this environment variable" 1>&2
+    exit 1
+fi
+
+export GOOSE_DBSTRING=postgres://${POSTGRES_USER}:${POSTGRES_PASSWORD}@localhost:${POSTGRES_PORT}/${POSTGRES_DATABASE}?sslmode=disable
+
+echo "Starting the cloudsql proxy"
+touch /cloudsql.log
+exec /proxy/cloud_sql_proxy -dir=/cloudsql -instances="$GCLOUD_SQL_INSTANCE" -credential_file=/sa.json > /cloudsql.log 2>&1 &
+
+# Set true in the shared memory
+# We use this to detect if the ready message was received or nto
+echo "1" >/dev/shm/cloudsql_ready
+
+# Wait for the cloudsql connection to go up
+echo "Waiting for ready message from cloudsql proxy..."
```

Comment:
I liked more docker-compose health-checks but fine, does docker itself has something for that or nop?